todo_app/data/trello_items.py

```diff
@@ -0,0 +1,68 @@
+import requests
+import os
+from todo_app.data.Item import Item
+
+
+def get_lists():
+    url = "https://api.trello.com/1/boards/" + os.getenv('BOARD_ID') + "/lists"
+    query = {'key': os.getenv('API_KEY'), 'token': os.getenv('TOKEN'), 'cards': 'open'}
+
+    response = (requests.get(url, params=query)).json()
+
+    return response
+
+
+def alterCardInfo(list, status):
+    cards = []
+    for card in list:
+        cards.append(Item.from_trello_card(card, status))
+    return cards
+
+
+def get_todo():
+    todolist = (get_lists())[0]['cards']
+    return alterCardInfo(todolist, 'Not Started')
+
+
+def get_done():
+    todolist = (get_lists())[1]['cards']
+    return alterCardInfo(todolist, 'Completed')
+
+
+def get_all_items():
+    return get_todo() + get_done()
+
+
+def create_card(title):
+    listid = (get_lists())[0]['id']
+    url = "https://api.trello.com/1/cards/"
+    query = {'key': os.getenv('API_KEY'), 'token': os.getenv('TOKEN'), 'name': title, 'idList': listid}
+
+    requests.post(url, params=query)
+
+
+def delete_card(item):
+    url = "https://api.trello.com/1/cards/" + item
+    query = {'key': os.getenv('API_KEY'), 'token': os.getenv('TOKEN')}
+    requests.delete(url, params=query)
+
+
+def update_card(item, query):
+    url = "https://api.trello.com/1/cards/" + item
+    requests.put(url, params=query)
+
+
+def complete_item(item):
+    update_card(item, {'key': os.getenv('API_KEY'), 'token': os.getenv('TOKEN'), 'idList': (get_lists())[1]['id']})
```

Comment:
Could we select the lists by name rather than index?

Or at least could we declare a variable with a name like `completed_list_id`?

Reply:
[Can we do the same for the undo_complete function too?]